pandas/tests/generic/test_finalize.py

```diff
@@ -762,7 +761,7 @@ def test_groupby_finalize(obj, method):
     [
         lambda x: x.agg(["sum", "count"]),
         lambda x: x.transform(lambda y: y),
-        lambda x: x.apply(lambda y: y),
+        # lambda x: x.apply(lambda y: y), Fixed with #42252
```

Comment:
I am not sure I understand what the changes in ``concat`` behavior have to do with this test.
Can you please clarify?

Reply:
In this case, the `apply` will use concat internally https://github.com/pandas-dev/pandas/blob/be2efa98dd97f86809d741070757fd1cbe425fc2/pandas/core/apply.py#L382.

Reply:
Yet, I am not sure I understood the reason.
Maybe some other reviewers will catch the idea.

Reply:
@ivanovmg Sorry for the confusion. The test checks if `apply` will retain the `attrs` and mark this as failing as the `attrs` should get lost.

However, in this test, the `apply` calls `concat`, which will now preserve the `attrs`. 
Thus, in this test, all the operations done will preserve the `attrs` and thus, the final result will have `attrs` and thus, the test will no longer fail. So I have removed the mark as fail.

Reply:
don't comment out cases